mrp_finished_backorder_product/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "MRP Finished Products",
```

Comment:
Change it to "MRP Finished Backorder Product" please :pray:

Reply:
Solved thank you!!